std/typecons.d

```diff
@@ -836,6 +844,26 @@ if (distinctFieldNames!(Specs))
             assert(tup1 > tup2);
         }
 
+        /// Allow tuple concatenation
+        auto opBinary(string op, T)(T t)
+        if (op == "~")
+        {
+            static if (isTuple!T)
+                return Tuple!(AliasSeq!(_Fields, T._Fields))(expand, t.expand);
```

Comment:
I think the use of `AliasSeq` here is redundant, but I'm not completely sure without checking it.